test/integration/navbar/nonuser_navbar_test.rb

```diff
@@ -15,29 +15,31 @@ class NonuserNavbar < ActionDispatch::IntegrationTest
     assert page.has_link? 'Home'
     find_link('Home', match: :first).click
     assert_equal '/', current_path
-    find_link('Discover Previous Conversations', match: :first).click
-    assert_equal '/supportourwork', current_path
-    assert page.has_link? 'Invite Us To Speak'
-    find_link('Invite Us To Speak', match: :first).click
+    assert page.has_link? 'About Us'
+    find_link('About Us', match: :first).click
+    assert_equal '/getinvolved', current_path
+    assert page.has_link? 'Have us on your Podcast'
+    find_link('Have us on your Podcast', match: :first).click
     assert_equal '/drschaeferspeaking', current_path
-    assert page.has_link? 'Be a ThinQtv Influencer!'
-    find_link('Be a ThinQtv Influencer!', match: :first).click
-    assert_equal '/internship', current_path
+    assert page.has_link? 'Join the Team'
+    find_link('Join the Team', match: :first).click
+    assert_equal '/jointheteam', current_path
+    assert page.has_link? 'Study Hall'
+    find_link('Study Hall', match: :first).click
+    assert_equal '/studyhall', current_path
     assert page.has_link? 'FAQ'
     find_link('FAQ', match: :first).click
     assert_equal '/faq', current_path
+    assert page.has_link? 'Terms of Service'
+    find_link('Terms of Service', match: :first).click
+    assert_equal '/tos', current_path
   end
 
   test 'sign up' do
     assert page.has_css? '.navbar-btn'
     assert page.has_link? 'Sign Up'
     find_link('Sign Up', match: :first).click
     assert_equal '/signup', current_path
-    assert_emails 2 do
```

Comment:
Tests are required to verify emails are sent

Reply:
Can an assertion test whether they're actually sent or can we only test whether they're queued - we don't have access to test gmail

Reply:
Anyway, sounds like this is not ready to be merged into master